src/test/kotlin/com/exactpro/th2/codec/ProcessorTest.kt

```diff
@@ -31,7 +31,7 @@ class ProcessorTest {
 
     @Test
     fun `simple test - decode`() {
-        val processor = DecodeProcessor(TestCodec(false), ORIGINAL_PROTOCOLS, CODEC_EVENT_ID.toProto()) { }
+        val processor = TransportDecodeProcessor(TestCodec(false), ORIGINAL_PROTOCOLS, CODEC_EVENT_ID.toProto()) { }
```

Comment:
Please save the test for both protocols. We can create parameterize test where processor and message transform method will be parts of argument